client/pages/admin/admin-page.js

```diff
@@ -26,7 +26,7 @@ Template.admin_page.helpers(helpers);
 Template.admin_page.helpers({
   isUserMemberOfOrganizationWithId,
   organizations() {
-    return Organizations.find({});
+    return Organizations.find({}, { sort: { name: 1 }});
```

Comment:
:D - is there also an index for this?

Reply:
There is, but this happens in minimongo on client side, so it is not used